samples/complex/copilotstudio-skill/donet/appsettings.json

```diff
@@ -0,0 +1,43 @@
+{
+  "AgentApplication": {
```

Comment:
is this section required in this sample?

Reply:
I believe all DotNet are using config for options.